.pypirc

```diff
@@ -0,0 +1,7 @@
+[distutils]
+index-servers = local
+
+[local]
+repository: https://otomato.jfrog.io/artifactory/api/pypi/PyPi
+username: candidate
+password: AP3dLQtysxAU27HAhoJAzUiGfYJ
```

Comment:
This is a security problem!
We never put passwords as clear unencrypted text into git.
Please reimplement using github secrets: https://docs.github.com/en/actions/reference/encrypted-secrets